### PR TITLE
unify image build

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

part of https://github.com/kubeedge/kubeedge/issues/3518
unify the image cmd like make binary as below
`make image WHAT=cloudcore`

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -307,33 +307,26 @@ ARCH ?= amd64
 IMAGE_TAG ?= $(shell git describe --tags)
 GO_LDFLAGS='$(shell hack/make-rules/version.sh)'
```
Comment (fisherxu) [inline Makefile:308]: Seems all these value can be removed?

**Comment by fisherxu** [comment]
cc @zhu733756

File: hack/make-rules/image.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  admission:admission:build/admission/Dockerfile
+  edgecore:edgecore:build/edge/Dockerfile
+  edgesite-agent:edgesite-agent:build/edgesite/agent-build.Dockerfile
+  edgesite-server:edgesite-server:build/edgesite/server-build.Dockerfile
+  csidriver:csidriver:build/csidriver/Dockerfile
+  iptablesmanager:iptables-manager:build/iptablesmanager/Dockerfile
+)
+
+GO_LDFLAGS="$(${KUBEEDGE_ROOT}/hack/make-rules/version.sh)"
+IMAGE_TAG=$(git describe --tags)
```
Comment (fisherxu) [inline hack/make-rules/image.sh:37]: Now the `IMAGE_TAG` is from `git describe --tags`, for daily build, what IMAGE_TAG should used? cc @zhu733756 @zc2638 Is the `git describe --tags` still ok?

File: Makefile (hunk 1 above)
Comment (gy95) [reply Makefile]: done, and now both helm_e2e and `make image` cmd use default image tag defined by cmd `shell git describe --tags`

File: hack/make-rules/image.sh (hunk 2 above)
Comment (zhu733756) [reply hack/make-rules/image.sh:37]: > Now the `IMAGE_TAG` is from `git describe --tags`, for daily build, what IMAGE_TAG should used? cc @zhu733756 @zc2638 Is the `git describe --tags` still ok?

I think it is ok for me.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Comment by zc2638** [comment]
If it is only used to generate a release version, then it is reasonable.
If it is compatible with cicd, it needs the current commit hash.
@fisherxu @gy95

**Pull request merged**